data/json/bionics.json

```
@@ -231,7 +224,8 @@
     "type": "bionic",
     "name": { "str": "Bionic Claws" },
     "description": "Vicious claws have been surgically installed inside your fingers, allowing you to extend and retract them at the cost of a small amount of power.  These do considerable cutting damage, but prevent you from holding anything else while extended.",
-    "occupied_bodyparts": [ [ "hand_l", 5 ], [ "hand_r", 5 ] ],
+    "occupied_bodyparts": [ [ "hand_l", 4 ], [ "hand_r", 4 ] ],
+    "encumbrance": [ [ "hand_l", 3 ], [ "hand_r", 3 ] ],
```

```suggestion
    "encumbrance": [ [ "hand_l", 1 ], [ "hand_r", 1 ] ],
```
When toggled, however, this should add at least 3 encumbrance to each; that will require modifications to bio_claws_weapon. (I have my doubts as to whether they'd be inside the fingers in the first place, or in the palm of the hand between the metacarpal bones.)

---

Changing encumbrance only with BIONIC_TOGGLED flag is possible?
If not I would just keep this 3 encumbrance

---

Should be - add encumbrance to the generated fake item.

---

The fake item is a weapon, not `type: ARMOR` with damage added.
There is no instance of not-armor weapons giving you encumbrance, so I guess it's not implemented.
I would keep bionic's encumbrance of 3.